examples/configs/nemotron/README.md

```diff
@@ -0,0 +1,107 @@
+```
+
+### Controlling Detailed Thinking in Final Responses
+
+As Nemotron is a hybrid reasoning model, users can toggle the "detailed thinking" feature for final responses similar to how it works on build.nvidia.com:
+
+#### Enabling Detailed Thinking via System Message
+
+To enable detailed thinking in the response, include a system message with "detailed thinking on":
+
+```python
+response = rails.generate(messages=[
+    {"role": "system", "content": "detailed thinking on"},
+    {"role": "user", "content": "How is the weather today?"}
+])
+```
+
+This will include the model's reasoning process in a `<think>...</think>` wrapper:
+
+```
+{'role': 'assistant',
+ 'content': '<think>\n</think>I\'m sorry, but I don\'t know the weather. I\'m a large language model, I don\'t have access to real-time information or your location. However, I can guide you on how to check the weather! You can check the weather forecast for your area by:...'}
+```
+
+#### Standard Mode (No Detailed Thinking)
+
+Without the special system message, the model provides direct responses without showing its reasoning:
+
+```python
+response = rails.generate(messages=[
+    {"role": "user", "content": "How is the weather today?"}
+])
+```
+
+Response:
+
+```
+{'role': 'assistant',
+ 'content': 'The weather! Unfortunately, I don\'t have real-time access to current weather conditions or your location. I\'m a large language model...'}
+```
+
+To remove the thinking traces from the internal tasks, you can use the `remove_thinking_traces` configuration option:
```

Comment:
Below, it's "remove_reasoning_traces" -- are they the same?

Reply:
yes

Reply:
but I will rename it to `remove_reasoning_traces` as `remove_thinking_traces` is deprecated (still backward compatible)